integration/substrate/release_version.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { createConnection, deploy, aliceKeypair, debug_buffer } from "./index";
+import expect from 'expect';
+import { ContractPromise } from "@polkadot/api-contract";
+
+describe('Deploy release_version.sol and test the debug buffer is empty', () => {
+    it('removes all debugging', async function () {
+        let conn = await createConnection();
+        const alice = aliceKeypair();
+        let deployed_contract = await deploy(
+            conn,
+            alice,
+            "release.contract",
+            BigInt(0)
+        );
+        let contract = new ContractPromise(
+            conn,
+            deployed_contract.abi,
+            deployed_contract.address
+        );
+
+        // The --release flag should remove all debugging features, making the debug buffer empty
+        let res = await debug_buffer(conn, contract, `print_then_error`, [20])
```

Comment:
Can you change this so it calls the contract twice:
- First time it does not overflow and we should check the result. The call should succeed, the debug buffer should be empty
- Second time it does overflow, we don't check the result but still expect the debug buffer to be empty.

The problem with the test as it is now is that the call might fail for any reason and leave the debug buffer empty (it doesn't really tell why the debug buffer was empty).

Reply:
But now the overflow flag is removed, meaning that the output will not change only if there is an `unchecked` block added to the contract. There is also a print statement that will be in the debug buffer if the contract fails